main.py

```
@@ -181,5 +182,64 @@ def clear_cache():
+    except RuntimeError as e:
+        raise click.ClickException(f"Runtime error: {e}") from e
+
+@cli.command()
+@click.argument('date', required=True)
+def build_digest_for_date(date):
+    """Build a JSON digest for a specific date."""
+    # Validate date format
+    try:
+        datetime.strptime(date, '%Y-%m-%d')
+    except ValueError as exc:
+        raise click.BadParameter("Date must be YYYY-MM-DD (e.g., 2025-01-15).") from exc
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)